### PR TITLE
Rusui Tier 2 mirror

### PR DESCRIPTION
**Pull request by KittyLe**
Yes. Very stable.
```
[ket@backup cyber]$ uptime
 06:13:44 up 18 days, 12:34,  1 user,  load average: 0.00, 0.00, 0.00
[ket@backup cyber]$ 
```

### CODE REVIEW

File: mirrorlist [hunk 1]
```diff
@@ -1,12 +1,15 @@
 ##
 ## CyberOS repository mirrorlist
-## Last updated: 2021-05-10
+## Last updated: 2021-06-10
```
Comment (omaemae) [inline mirrorlist:3]: what

**Review by omaemae — CHANGES_REQUESTED** [review]

File: mirrorlist (hunk 1 above)
Comment (KittyLe) [reply mirrorlist:3]: I'm bad at calendar

**Review by KittyLe — COMMENTED** [review]
bro fixe

**Review by omaemae — APPROVED** [review]
LGTM, hope it builds

**Comment by omaemae** [comment]
wait how do I merge from mobile this UI makes no sense

**Pull request merged**